src/output.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Output helpers.
+
+// COLORS:
+// The output colour of a particular object of text should convey some information about what
+// exactly it is. These are not hard and fast rules, but generally speaking:
+// - Blue for general information.
+// - Green for success.
+// - Red for errors and warnings.
+// - Cyan should be used for section titles.
+// - Yellow should represent field names -- i.e. UUIDs, versions, etc.
+//
+// By sticking to this guideline we ensure that the CLI stays consistent.
```

Comment:
Agree with Ionut that it would be nice to have macros mapping with each one of those colours to be consistent everywhere. Also, it would maybe be nice to not have the `macro_export` but let the module where they are needed import them directly like static functions.